decidim-budgets/config/locales/es.yml

```diff
@@ -53,7 +53,7 @@ es:
           title: Confirmar voto
         budget_excess:
           close: Cerrar
-          description: Este proyecto sobrepasa el presupuesto máximo y no se puede añadir. Si lo deseas, puede eliminar un proyecto que ya hayas seleccionado o votar con tus preferencias.
+          description: Este proyecto sobrepasa el presupuesto máximo y no se puede añadir. Si lo deseas, puede eliminar un proyecto que ya hayas seleccionado o dar apoyo con tus preferencias.
```

Comment:
`s/puede/puedes/`
Also, `o apoyarle`, removing the `con tus preferencias` part, because `day apoyo` assumes voting in favor. 

That's why I don't like the `apoyar` use when voting, because you can vote negatively, but `apoyar` assumes a favorable vote.